src/main/webapp/onelive.html

```diff
@@ -49,11 +273,82 @@
 <!-- Argon JS -->
 <script src="assets/js/argon.js?v=1.0.1"></script>
 <script src="assets/js/global-custom-js.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+
 <!-- Script for static navbar and footer -->
 <script src="assets/js/navigation.js"></script>
+
+<script type="text/html" id="templateTeam">
+    <div class="row">
+        {{#data}}
+        <div class="col-md-3 col-lg-3 mb-5">
+            <div class="px-4">
+                <div class="imgContainer">
+                    <img src="{{image}}"
+                         class="rounded-circle img-center img-fluid shadow shadow-lg--hover imgHover"
+                    >
+                    <div class="imgIcon">
+                        <a href="{{linkedin}}" class="btn btn-default btn-icon-only rounded-circle">
+                            <i class="fa fa-linkedin"></i>
+                        </a>
+                    </div>
+                </div>
+                <div class="pt-4 text-center">
+                    <h5 class="title">
+                        <span class="d-block mb-1">{{name}}</span>
+                        <small class="h6 text-muted">{{city}}</small>
+                    </h5>
+                </div>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
+
 <script>
     $(function () {
-        loadNavAndFooter('assets/content/static')  //relative path to content directory
-    })
+        loadNavAndFooter('assets/content');  //relative path to content directory
+        loadProfiles(); // call function to load profiles
+    });
+
+    //function to load profile data
+    function loadProfiles() {
+        $.ajax({
+            type: 'get',
+            url: 'assets/content/json/v3.0.0/onelive.json',
+            dataType: 'json',
+            success: function (data) {
+
```

Comment:
Use the updated function which has animated loading (check https://github.com/sef-global/sef-site/blob/master/src/main/webapp/examples/team.html )

Reply:
Done!